mathesar/api/serializers/functions.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+
+
+class DbFunctionSerializer(serializers.Serializer):
+    id = serializers.CharField()
```

Comment:
You'll need to also include a list of database IDs that the function is available on. Not all functions will be installed or available on all databases.

Reply:
It looks like you've instead chosen to only make these available on a single database. In that case, they should not be a top level resource, please see my comment below. If we do choose to make them available on multiple databases, then you'll need to specify the database ID in the serializer.